uportal-war/src/main/java/org/jasig/portal/rest/PortletsRESTController.java

```diff
@@ -55,24 +62,22 @@
 @Controller
 public class PortletsRESTController {
 
+    @Autowired
     private IPortletDefinitionRegistry portletDefinitionRegistry;
+
+    @Autowired
     private IPortletCategoryRegistry portletCategoryRegistry;
-    private IPersonManager personManager;
 
     @Autowired
-    public void setPortletDefinitionRegistry(IPortletDefinitionRegistry portletDefinitionRegistry) {
```

Comment:
When you go to write the JUnit test for this controller, are you going to wish there were these `public` setter methods intact so that you can inject Mockito mocks for the portlet definition registry and the person manager?